zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/ElasticsearchEngineClient.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.schema;
+
+import co.elastic.clients.elasticsearch.ElasticsearchClient;
+import co.elastic.clients.elasticsearch._types.mapping.TypeMapping;
+import co.elastic.clients.elasticsearch.cluster.PutComponentTemplateRequest;
+import co.elastic.clients.elasticsearch.indices.Alias;
+import co.elastic.clients.elasticsearch.indices.CreateIndexRequest;
+import co.elastic.clients.elasticsearch.indices.IndexTemplateSummary;
+import co.elastic.clients.elasticsearch.indices.PutIndexTemplateRequest;
+import co.elastic.clients.elasticsearch.indices.PutMappingRequest;
+import co.elastic.clients.json.JsonpDeserializer;
+import io.camunda.exporter.exceptions.ExporterElasticsearchException;
+import io.camunda.exporter.schema.descriptors.ComponentTemplateDescriptor;
+import io.camunda.exporter.schema.descriptors.IndexDescriptor;
+import io.camunda.exporter.schema.descriptors.IndexTemplateDescriptor;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ElasticsearchEngineClient implements SearchEngineClient {
+  private final ElasticsearchClient client;
+  private final Logger log = LoggerFactory.getLogger(ElasticsearchEngineClient.class);
+
+  public ElasticsearchEngineClient(final ElasticsearchClient client) {
+    this.client = client;
+  }
+
+  @Override
+  public void createIndex(final IndexDescriptor indexDescriptor) {
+    final CreateIndexRequest request = createIndexRequest(indexDescriptor);
+    try {
+      client.indices().create(request);
+      log.debug("Index [{}] was successfully created", indexDescriptor.getIndexName());
+    } catch (final IOException e) {
+      log.error("Index [{}] was NOT created", indexDescriptor.getIndexName(), e);
+      throw new ExporterElasticsearchException(e.getMessage());
```

Comment:
I'd pass the custom message logged above as message, and the throwable.
```
      throw new ElasticsearchExporterException("Index [{}] was NOT created", e);

```

you shall use `String.format` to construct the message with the indexDescriptor.getIndexName() included